### ISSUE TITLE
reference number

### ISSUE THREAD

**Issue by edgartdata** [PHAROS-photoarchives/Zeri-Foundation#4]
line 28: not sure what this is for?

**Comment by marilenadaquino** [comment]
Is the inventory number of the negative. Whatever they catalog, whether is a positive, a copy etc. if they find (e.g. in a photographer's catalogue) the number of the original negative they record it there. This could be useful - in few cases I guess - for reconciling same pictures owned by the different archives.

**Comment by edgartdata** [comment]
Yes, we said we were interested in keeping track of the inventory number of the negative for exactly that purpose.

**Comment by edgartdata** [comment]
Do you see in line 6 that Anastasia notes that the Frick only records the negative number and use it as the unique identifier for the photo? So I guess they will need to repeat that id in line 28 as well?

**Comment by marilenadaquino** [comment]
yup. if the two ids overlap they should keep both in different fields. 
In Zeri the negative number is not always available, hence they have a different identifier.
